docs/sources/administration/user-management/_index.md

```diff
@@ -13,6 +13,9 @@ weight: 200
 
 A _user_ is defined as any individual who can log in to Grafana. Each user is associated with a _role_ that includes _permissions_. Permissions determine the tasks a user can perform in the system. For example, the **Admin** role includes permissions for an administrator to create and delete users.
 
+Watch this short video to get started. 
```

Comment:
The video in this PR looks appropriate for this page.

Instead of this short sentence, would you be able to more detail that describes the contents of the video. I think that will make this addition more accessible and also help readers decide on whether they want to watch the video. or continue with the documentation without watching.

Reply:
Hey @jdbaldry! Sorry I didn't see the notification message until now!  

What do you think about adding "Watch this video to learn how to manage users and permissions in Grafana OSS and Grafana Cloud"?

Reply:
Sorry for the delay, I was out of office on Friday.

I think that gives a little more context than before which is useful. 

I was wondering if it is worth mentioning all the topics in the video like in the YouTube video description but I think that list may affect the scannability of the page.

```suggestion
Watch the following video to learn how to manage users and permissions in Grafana OSS and Grafana Cloud:
```